tests/Unit/NumericalAlgorithms/Interpolation/Test_InterpolateToTarget.cpp

```diff
@@ -77,23 +77,63 @@ struct initialize_elements_and_queue_simple_actions {
     for (const auto& element_id : element_ids) {
       ActionTesting::next_action<elem_component>(make_not_null(&runner),
                                                  element_id);
+      if (not Metavariables::should_interpolate_value and
+          Metavariables::has_should_interpolate) {
```

Comment:
Just a question: shouldn't `has_should_interpolate` be checked before `should_interpolate_value`?  It doesn't matter for this test because `should_interpolate_value` is always defined even if `has_should_interpolate` is false, but the ordering struck me as strange.